### PR TITLE
Add build script for libseccomp-golang

### PR DESCRIPTION
**Pull request by Chandranana**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
```
Comment (seth-priya) [inline l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh:22]: this is already set in the first line, no need to repeat

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    echo "------------------$PACKAGE_NAME:test_fails---------------------"
+	echo "$PACKAGE_NAME  | $OS_NAME | GitHub | Fail |  Test_Fails"
+	exit 1
+else
+	echo "------------------$PACKAGE_NAME:install_and_test_success-------------------------"
+	echo "$PACKAGE_NAME | $OS_NAME | GitHub | Pass |  Install_and_Test_Success"
+	exit 0
+fi
+
+bash
```
Comment (seth-priya) [inline l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh:55]: the next 3 lines aren't probably needed? any specific reason for including?

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+tar -C /bin -xf go1.19.linux-ppc64le.tar.gz 
+mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
+rm -rf go1.19.linux-ppc64le.tar.gz
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/tester/go
+export PATH=$GOPATH/bin:$PATH
+export GO111MODULE=on
+
+#Download source for libseccomp-golang 
+cd /root
```
Comment (seth-priya) [inline l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh:41]: let's not hard code this, define/set WORKDIR

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh (hunk 2 above)
Comment (Chandranana) [reply l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh]: No,not needed. I had copied it as per reference script. Removed the same.

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh (hunk 3 above)
Comment (Chandranana) [reply l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh]: Done

**Comment by Chandranana** [comment]
@seth-priya Updated the code changes, please check.

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=${2:-libseccomp-golang}
```
Comment (seth-priya) [inline l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh:22]: this one should actually be a constant else the script will fail

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
Please remove the log files from the PR @Chandranana

**Comment by Chandranana** [comment]
> Please remove the log files from the PR @Chandranana

ok

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh (hunk 4 above)
Comment (Chandranana) [reply l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh:22]: @seth-priya I was referring to gofuzz_ubi_8.5.sh script here it is defined likewise.

**Comment by Chandranana** [comment]
> Please remove the log files from the PR @Chandranana

@seth-priya This is done, kindly check

File: l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh (hunk 4 above)
Comment (Chandranana) [reply l/libseccomp-golang/libseccomp-golang_ubi_8.5.sh]: Done

**Comment by seth-priya** [comment]
Please make the PACKAGE-VERSION configurable with a default value

**Comment by Chandranana** [comment]
> Please make the PACKAGE-VERSION configurable with a default value

Done. Please check.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**